js/serial.js

```diff
@@ -18,18 +18,11 @@ var serial = {
     logHead: 'SERIAL: ',
 
     connect: function (path, options, callback) {
-        var self = this;
```

Comment:
Thanks. In terms of maintainability, making a distinction between short functions and long functions is a fallacy, since most of the time long functions start as short functions, and become long functions after people add functionality. When it comes to taking self references in JS, I try to do it consistently, since doing so without need doesn't break anything, but not using a self reference and then refactoring code to be inside a callback leads to often hard to find errors.